[2025-01-19 20:59:15,621: INFO: main: Welcome to the custom logging data science]
[2025-01-20 12:02:35,890: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-01-20 12:03:57,028: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-01-20 12:03:57,038: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-20 12:03:57,040: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-01-20 12:03:57,042: INFO: common: created directory at: artifacts]
[2025-01-20 12:03:57,043: INFO: common: created directory at: artifacts/data_ingestion]
[2025-01-20 12:03:57,975: INFO: 2662890588: artifacts/data_ingestion/data.zip download! with following info: 
Connection: close
Content-Length: 23329
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "c69888a4ae59bc5a893392785a938ccd4937981c06ba8a9d6a21aa52b4ab5b6e"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 8CB3:08D8:2A734D:3743D2:678E65DD
Accept-Ranges: bytes
Date: Mon, 20 Jan 2025 15:03:57 GMT
Via: 1.1 varnish
X-Served-By: cache-scl2220028-SCL
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1737385437.484591,VS0,VE332
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 12448feecda00516a97f87c9fb603d8a8fddb7e4
Expires: Mon, 20 Jan 2025 15:08:57 GMT
Source-Age: 0

]
[2025-01-20 16:33:37,367: INFO: main: Welcome to the custom logging data science]
[2025-01-20 16:33:37,368: INFO: main: >>>> stage Data Ingestion stage started <<<<<]
[2025-01-20 16:33:37,383: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-01-20 16:33:37,394: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-20 16:33:37,397: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-01-20 16:33:37,398: INFO: common: created directory at: artifacts]
[2025-01-20 16:33:37,399: INFO: common: created directory at: artifacts/data_ingestion]
[2025-01-20 16:33:37,400: INFO: data_ingestion: File already exists]
[2025-01-20 16:33:37,419: INFO: main: >>>>> stage Data Ingestion stage completed

x=============x]
[2025-01-20 16:33:59,287: INFO: main: Welcome to the custom logging data science]
[2025-01-20 16:33:59,288: INFO: main: >>>> stage Data Ingestion stage started <<<<<]
[2025-01-20 16:33:59,289: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-01-20 16:33:59,290: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-20 16:33:59,291: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-01-20 16:33:59,292: INFO: common: created directory at: artifacts]
[2025-01-20 16:33:59,294: INFO: common: created directory at: artifacts/data_ingestion]
[2025-01-20 16:34:00,270: INFO: data_ingestion: artifacts/data_ingestion/data.zip download! with following info: 
Connection: close
Content-Length: 23329
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "c69888a4ae59bc5a893392785a938ccd4937981c06ba8a9d6a21aa52b4ab5b6e"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 6569:123197:2E0EF:36F4E:678EA526
Accept-Ranges: bytes
Date: Mon, 20 Jan 2025 19:33:59 GMT
Via: 1.1 varnish
X-Served-By: cache-scl2220027-SCL
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1737401640.594770,VS0,VE236
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: d38911988391dc0517ce3c7be473efe4afb7990e
Expires: Mon, 20 Jan 2025 19:38:59 GMT
Source-Age: 0

]
[2025-01-20 16:34:00,286: INFO: main: >>>>> stage Data Ingestion stage completed

x=============x]
[2025-01-21 11:10:06,485: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-01-21 11:10:06,489: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-21 11:10:06,497: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-01-21 11:10:06,499: INFO: common: created directory at: artifacts]
[2025-01-21 11:10:06,500: INFO: common: created directory at: artifacts/data_validation]
[2025-01-21 12:34:52,559: INFO: main: Welcome to the custom logging data science]
[2025-01-21 12:34:52,559: INFO: main: >>>> stage Data Ingestion stage started <<<<<]
[2025-01-21 12:34:52,560: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-01-21 12:34:52,561: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-21 12:34:52,562: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-01-21 12:34:52,563: INFO: common: created directory at: artifacts]
[2025-01-21 12:34:52,563: ERROR: main: 'ConfigurationManager' object has no attribute 'get_data_ingestion_config']
Traceback (most recent call last):
  File "C:\Users\roser\OneDrive\Documentos\endto\datascienceproject\main.py", line 14, in <module>
    data_ingestion.initiate_data_ingestion()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "C:\Users\roser\OneDrive\Documentos\endto\datascienceproject\src\datascience\pipeline\data_ingestion_pipeline.py", line 13, in initiate_data_ingestion
    data_ingestion_config=config.get_data_ingestion_config()
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'ConfigurationManager' object has no attribute 'get_data_ingestion_config'. Did you mean: 'get_data_validation_config'?
[2025-01-21 12:35:25,651: INFO: main: Welcome to the custom logging data science]
[2025-01-21 12:35:25,651: INFO: main: >>>> stage Data Ingestion stage started <<<<<]
[2025-01-21 12:35:25,652: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-01-21 12:35:25,653: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-21 12:35:25,654: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-01-21 12:35:25,660: INFO: common: created directory at: artifacts]
[2025-01-21 12:35:25,661: INFO: common: created directory at: artifacts/data_validation]
[2025-01-21 12:35:25,661: ERROR: main: 'DataValidationConfig' object has no attribute 'local_data_file']
Traceback (most recent call last):
  File "C:\Users\roser\OneDrive\Documentos\endto\datascienceproject\main.py", line 14, in <module>
    data_ingestion.initiate_data_ingestion()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "C:\Users\roser\OneDrive\Documentos\endto\datascienceproject\src\datascience\pipeline\data_ingestion_pipeline.py", line 15, in initiate_data_ingestion
    data_ingestion.download_file()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "C:\Users\roser\OneDrive\Documentos\endto\datascienceproject\src\datascience\components\data_ingestion.py", line 12, in download_file
    if not os.path.exists(self.config.local_data_file):
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'DataValidationConfig' object has no attribute 'local_data_file'
[2025-01-21 12:45:43,984: INFO: main: Welcome to the custom logging data science]
[2025-01-21 12:45:43,985: INFO: main: >>>> stage Data Ingestion stage started <<<<<]
[2025-01-21 12:45:43,986: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-01-21 12:45:43,987: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-21 12:45:43,988: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-01-21 12:45:43,988: INFO: common: created directory at: artifacts]
[2025-01-21 12:45:43,988: ERROR: main: 'ConfigurationManager' object has no attribute 'get_data_ingestion_config']
Traceback (most recent call last):
  File "C:\Users\roser\OneDrive\Documentos\endto\datascienceproject\main.py", line 14, in <module>
    data_ingestion.initiate_data_ingestion()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "C:\Users\roser\OneDrive\Documentos\endto\datascienceproject\src\datascience\pipeline\data_ingestion_pipeline.py", line 13, in initiate_data_ingestion
    data_ingestion_config=config.get_data_ingestion_config()
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'ConfigurationManager' object has no attribute 'get_data_ingestion_config'. Did you mean: 'get_data_validation_config'?
[2025-01-21 12:46:44,008: INFO: main: Welcome to the custom logging data science]
[2025-01-21 12:46:44,008: INFO: main: >>>> stage Data Ingestion stage started <<<<<]
[2025-01-21 12:46:44,010: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-01-21 12:46:44,010: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-21 12:46:44,011: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-01-21 12:46:44,012: INFO: common: created directory at: artifacts]
[2025-01-21 12:46:44,012: INFO: common: created directory at: artifacts/data_ingestion]
[2025-01-21 12:46:44,013: INFO: data_ingestion: File already exists]
[2025-01-21 12:46:44,041: INFO: main: >>>>> stage Data Ingestion stage completed

x=============x]
[2025-01-21 12:46:44,041: INFO: main: >>>> stage Model Evaluation stage started <<<<<]
[2025-01-21 12:46:44,043: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-01-21 12:46:44,043: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-21 12:46:44,044: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-01-21 12:46:44,045: INFO: common: created directory at: artifacts]
[2025-01-21 12:46:44,046: INFO: common: created directory at: artifacts/data_validation]
[2025-01-21 12:46:44,073: INFO: main: >>>>> stage Model Evaluation stage completed

x=============x]
[2025-01-21 15:59:09,898: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-01-21 15:59:09,900: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-21 15:59:09,903: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-01-21 15:59:09,905: INFO: common: created directory at: artifacts]
[2025-01-21 15:59:09,906: INFO: common: created directory at: artifacts/data_transformation]
[2025-01-21 16:00:03,688: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-01-21 16:00:03,689: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-21 16:00:03,691: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-01-21 16:00:03,692: INFO: common: created directory at: artifacts]
[2025-01-21 16:00:03,692: INFO: common: created directory at: artifacts/data_transformation]
[2025-01-21 16:00:03,709: INFO: 1586305886: splited data into training and test sets]
[2025-01-21 16:00:03,710: INFO: 1586305886: (1199, 12)]
[2025-01-21 16:00:03,710: INFO: 1586305886: (400, 12)]
[2025-01-21 16:19:04,386: INFO: main: Welcome to the custom logging data science]
[2025-01-21 16:19:04,386: INFO: main: >>>> stage Data Ingestion stage started <<<<<]
[2025-01-21 16:19:04,387: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-01-21 16:19:04,388: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-21 16:19:04,389: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-01-21 16:19:04,390: INFO: common: created directory at: artifacts]
[2025-01-21 16:19:04,391: INFO: common: created directory at: artifacts/data_ingestion]
[2025-01-21 16:19:04,391: INFO: data_ingestion: File already exists]
[2025-01-21 16:19:04,396: INFO: main: >>>>> stage Data Ingestion stage completed

x=============x]
[2025-01-21 16:19:04,396: INFO: main: >>>> stage Model Evaluation stage started <<<<<]
[2025-01-21 16:19:04,399: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-01-21 16:19:04,399: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-21 16:19:04,400: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-01-21 16:19:04,401: INFO: common: created directory at: artifacts]
[2025-01-21 16:19:04,401: INFO: common: created directory at: artifacts/data_validation]
[2025-01-21 16:19:04,423: INFO: main: >>>>> stage Model Evaluation stage completed

x=============x]
[2025-01-21 16:19:04,424: INFO: main: >>>> stage Data Transformation stage started <<<<<]
[2025-01-21 16:19:04,436: INFO: main: >>>>> stage Data Transformation stage completed

x=============x]
[2025-01-21 16:20:00,306: INFO: main: Welcome to the custom logging data science]
[2025-01-21 16:20:00,307: INFO: main: >>>> stage Data Ingestion stage started <<<<<]
[2025-01-21 16:20:00,309: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-01-21 16:20:00,310: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-21 16:20:00,311: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-01-21 16:20:00,312: INFO: common: created directory at: artifacts]
[2025-01-21 16:20:00,313: INFO: common: created directory at: artifacts/data_ingestion]
[2025-01-21 16:20:00,313: INFO: data_ingestion: File already exists]
[2025-01-21 16:20:00,317: INFO: main: >>>>> stage Data Ingestion stage completed

x=============x]
[2025-01-21 16:20:00,317: INFO: main: >>>> stage Model Evaluation stage started <<<<<]
[2025-01-21 16:20:00,321: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-01-21 16:20:00,323: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-21 16:20:00,324: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-01-21 16:20:00,326: INFO: common: created directory at: artifacts]
[2025-01-21 16:20:00,326: INFO: common: created directory at: artifacts/data_validation]
[2025-01-21 16:20:00,355: INFO: main: >>>>> stage Model Evaluation stage completed

x=============x]
[2025-01-21 16:20:00,355: INFO: main: >>>> stage Data Transformation stage started <<<<<]
[2025-01-21 16:20:00,357: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-01-21 16:20:00,357: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-21 16:20:00,359: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-01-21 16:20:00,360: INFO: common: created directory at: artifacts]
[2025-01-21 16:20:00,360: INFO: common: created directory at: artifacts/data_transformation]
[2025-01-21 16:20:00,377: INFO: data_transformation: splited data into training and test sets]
[2025-01-21 16:20:00,377: INFO: data_transformation: (1199, 12)]
[2025-01-21 16:20:00,378: INFO: data_transformation: (400, 12)]
[2025-01-21 16:20:00,379: INFO: main: >>>>> stage Data Transformation stage completed

x=============x]
[2025-01-21 17:48:54,908: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-01-21 17:48:54,921: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-21 17:48:54,926: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-01-21 17:48:54,930: INFO: common: created directory at: artifacts]
[2025-01-21 17:49:16,854: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-01-21 17:49:16,855: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-21 17:49:16,857: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-01-21 17:49:16,859: INFO: common: created directory at: artifacts]
[2025-01-21 17:49:16,860: INFO: common: created directory at: artifacts]
[2025-01-21 17:49:53,706: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-01-21 17:49:53,708: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-21 17:49:53,711: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-01-21 17:49:53,713: INFO: common: created directory at: artifacts]
[2025-01-21 17:49:53,714: INFO: common: created directory at: artifacts]
[2025-01-21 17:50:06,975: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-01-21 17:50:06,979: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-21 17:50:06,982: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-01-21 17:50:06,983: INFO: common: created directory at: artifacts]
[2025-01-21 17:50:06,984: INFO: common: created directory at: artifacts]
[2025-01-21 17:50:48,968: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-01-21 17:50:48,971: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-21 17:50:48,974: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-01-21 17:50:48,975: INFO: common: created directory at: artifacts]
[2025-01-21 17:50:48,976: INFO: common: created directory at: artifacts]
[2025-01-21 17:51:20,092: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-01-21 17:51:20,094: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-21 17:51:20,097: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-01-21 17:51:20,099: INFO: common: created directory at: artifacts]
[2025-01-21 17:51:20,100: INFO: common: created directory at: artifacts/model_trainer]
[2025-01-21 18:01:43,900: INFO: main: Welcome to the custom logging data science]
[2025-01-21 18:01:43,900: INFO: main: >>>> stage Data Ingestion stage started <<<<<]
[2025-01-21 18:01:43,902: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-01-21 18:01:43,902: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-21 18:01:43,904: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-01-21 18:01:43,904: INFO: common: created directory at: artifacts]
[2025-01-21 18:01:43,904: INFO: common: created directory at: artifacts/data_ingestion]
[2025-01-21 18:01:43,905: INFO: data_ingestion: File already exists]
[2025-01-21 18:01:43,909: INFO: main: >>>>> stage Data Ingestion stage completed

x=============x]
[2025-01-21 18:01:43,909: INFO: main: >>>> stage Model Evaluation stage started <<<<<]
[2025-01-21 18:01:43,911: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-01-21 18:01:43,912: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-21 18:01:43,913: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-01-21 18:01:43,914: INFO: common: created directory at: artifacts]
[2025-01-21 18:01:43,914: INFO: common: created directory at: artifacts/data_validation]
[2025-01-21 18:01:43,936: INFO: main: >>>>> stage Model Evaluation stage completed

x=============x]
[2025-01-21 18:01:43,936: INFO: main: >>>> stage Data Transformation stage started <<<<<]
[2025-01-21 18:01:43,947: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-01-21 18:01:43,948: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-21 18:01:43,949: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-01-21 18:01:43,950: INFO: common: created directory at: artifacts]
[2025-01-21 18:01:43,950: INFO: common: created directory at: artifacts/data_transformation]
[2025-01-21 18:01:43,964: INFO: data_transformation: splited data into training and test sets]
[2025-01-21 18:01:43,965: INFO: data_transformation: (1199, 12)]
[2025-01-21 18:01:43,965: INFO: data_transformation: (400, 12)]
[2025-01-21 18:01:43,965: INFO: main: >>>>> stage Data Transformation stage completed

x=============x]
[2025-01-21 18:01:43,965: INFO: main: >>>> stage Model Trainer stage started <<<<<]
[2025-01-21 18:01:43,965: ERROR: main: 'ModelTrainderPipeline' object has no attribute 'initiate_data_transformation']
Traceback (most recent call last):
  File "C:\Users\roser\OneDrive\Documentos\endto\datascienceproject\main.py", line 51, in <module>
    data_ingestion.initiate_data_transformation()
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'ModelTrainderPipeline' object has no attribute 'initiate_data_transformation'
[2025-01-21 18:02:00,249: INFO: main: Welcome to the custom logging data science]
[2025-01-21 18:02:00,249: INFO: main: >>>> stage Data Ingestion stage started <<<<<]
[2025-01-21 18:02:00,250: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-01-21 18:02:00,251: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-21 18:02:00,252: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-01-21 18:02:00,252: INFO: common: created directory at: artifacts]
[2025-01-21 18:02:00,253: INFO: common: created directory at: artifacts/data_ingestion]
[2025-01-21 18:02:00,253: INFO: data_ingestion: File already exists]
[2025-01-21 18:02:00,255: INFO: main: >>>>> stage Data Ingestion stage completed

x=============x]
[2025-01-21 18:02:00,256: INFO: main: >>>> stage Model Evaluation stage started <<<<<]
[2025-01-21 18:02:00,257: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-01-21 18:02:00,258: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-21 18:02:00,260: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-01-21 18:02:00,260: INFO: common: created directory at: artifacts]
[2025-01-21 18:02:00,261: INFO: common: created directory at: artifacts/data_validation]
[2025-01-21 18:02:00,285: INFO: main: >>>>> stage Model Evaluation stage completed

x=============x]
[2025-01-21 18:02:00,286: INFO: main: >>>> stage Data Transformation stage started <<<<<]
[2025-01-21 18:02:00,288: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-01-21 18:02:00,289: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-21 18:02:00,290: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-01-21 18:02:00,290: INFO: common: created directory at: artifacts]
[2025-01-21 18:02:00,291: INFO: common: created directory at: artifacts/data_transformation]
[2025-01-21 18:02:00,310: INFO: data_transformation: splited data into training and test sets]
[2025-01-21 18:02:00,311: INFO: data_transformation: (1199, 12)]
[2025-01-21 18:02:00,311: INFO: data_transformation: (400, 12)]
[2025-01-21 18:02:00,312: INFO: main: >>>>> stage Data Transformation stage completed

x=============x]
[2025-01-21 18:02:00,312: INFO: main: >>>> stage Model Trainer stage started <<<<<]
[2025-01-21 18:02:00,314: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-01-21 18:02:00,316: INFO: common: yaml file: params.yaml loaded successfully]
[2025-01-21 18:02:00,318: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-01-21 18:02:00,318: INFO: common: created directory at: artifacts]
[2025-01-21 18:02:00,319: INFO: common: created directory at: artifacts/model_trainer]
[2025-01-21 18:02:00,387: INFO: main: >>>>> stage Model Trainer stage completed

x=============x]
